### PR TITLE
Refactor FXIOS-10205 [Swiftlint] Resolve 1 implicitly_unwrapped_optional violations in BottomSheetViewController

### PR DESCRIPTION
**Pull request by tonell-m**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-10205)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/22337)

## :bulb: Description

Continuing resolving `implicitly_unwrapped_optional` violations in order to enable the rule for further developments.

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [x] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

File: BrowserKit/Sources/ComponentLibrary/BottomSheet/BottomSheetViewController.swift [hunk 1]
```diff
@@ -175,7 +175,9 @@ public class BottomSheetViewController: UIViewController,
         view.addSubviews(dimmedBackgroundView, topTapView, sheetView)
         view.accessibilityElements = [closeButton, sheetView]
 
-        contentViewBottomConstraint = sheetView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        let contentViewBottomConstraint = sheetView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        self.contentViewBottomConstraint = contentViewBottomConstraint
```
Comment (Cramsden) [inline BrowserKit/Sources/ComponentLibrary/BottomSheet/BottomSheetViewController.swift:179]: Is this change needed as part of the unwrapped optional change? Couldn't we continue to set the classes property directly without needing to create an intermediate property?

**Review by Cramsden — COMMENTED** [review]
Hi 👋  Thank you so much for this contribution! This code looks good to me I just had one small question. Thanks again for your help.

File: BrowserKit/Sources/ComponentLibrary/BottomSheet/BottomSheetViewController.swift (hunk 1 above)
Comment (Cramsden) [reply BrowserKit/Sources/ComponentLibrary/BottomSheet/BottomSheetViewController.swift:179]: Ah I see. That makes sense!

**Review by Cramsden — APPROVED** [review]

File: BrowserKit/Sources/ComponentLibrary/BottomSheet/BottomSheetViewController.swift (hunk 1 above)
Comment (tonell-m) [reply BrowserKit/Sources/ComponentLibrary/BottomSheet/BottomSheetViewController.swift]: Hi @Cramsden, as suggested by @PARAIPAN9 in #23833 it's actually more consistent with the rest of the codebase to just activate the newly created constraint in place after initialising it as a class property instead of using a temporary local property

**Review by PARAIPAN9 — APPROVED** [review]
@tonell-m, great job and thanks for addressing the change!!
